tools/failures.py

```diff
@@ -53,123 +53,9 @@ def format_in_table(active_jobs, master):
-                 'plain-reftest-10': {'group': 'R', 'code': 'R10'},
-                 'plain-reftest-11': {'group': 'R', 'code': 'R11'},
-                 'plain-reftest-12': {'group': 'R', 'code': 'R12'},
-                 'plain-reftest-13': {'group': 'R', 'code': 'R13'},
-                 'plain-reftest-14': {'group': 'R', 'code': 'R14'},
-                 'plain-reftest-15': {'group': 'R', 'code': 'R15'},
-                 'plain-reftest-16': {'group': 'R', 'code': 'R16'},
-                 'plain-reftest-17': {'group': 'R', 'code': 'R17'},
-                 'plain-reftest-18': {'group': 'R', 'code': 'R18'},
-                 'plain-reftest-19': {'group': 'R', 'code': 'R19'},
-                 'plain-reftest-20': {'group': 'R', 'code': 'R20'},
-                 'reftest-1': {'group': 'R', 'code': 'R1'},
-                 'reftest-2': {'group': 'R', 'code': 'R2'},
-                 'reftest-3': {'group': 'R', 'code': 'R3'},
-                 'reftest-4': {'group': 'R', 'code': 'R4'},
-                 'reftest-e10s': {'group': 'R-e10s', 'code': 'R'},
-                 'reftest-e10s-1': {'group': 'R-e10s', 'code': 'R1'},
-                 'reftest-e10s-2': {'group': 'R-e10s', 'code': 'R2'},
-                 'crashtest-e10s': {'group': 'R-e10s', 'code': 'C'},
-                 'jittest': {'group': '', 'code': 'Jit'},
-                 'jittest-2': {'group': '', 'code': 'Jit2'},
-                 'jittest-1': {'group': '', 'code': 'Jit1'},
-                 'marionette': {'group': '', 'code': 'Mn'},
-                 'marionette-e10s': {'group': '', 'code': 'Mn-e10s'},
-                 'cppunit': {'group': '', 'code': 'Cpp'},
-                 'reftest-no-accel': {'group': '', 'code': 'Cpp'},
-                 'web-platform-tests-1': {'group': 'W', 'code': '1'},
-                 'web-platform-tests-2': {'group': 'W', 'code': '2'},
-                 'web-platform-tests-3': {'group': 'W', 'code': '3'},
-                 'web-platform-tests-4': {'group': 'W', 'code': '4'},
-                 'web-platform-tests-reftests': {'group': 'W', 'code': 'R'},
-                 'mochitest-jetpack': {'group': '', 'code': 'JP'},
-                 'luciddream': {'group': '', 'code': 'Ld'},
-                 'robocop-1': {'group': 'RC', 'code': '1'},
-                 'robocop-2': {'group': 'RC', 'code': '2'},
-                 'robocop-3': {'group': 'RC', 'code': '3'},
-                 'robocop-4': {'group': 'RC', 'code': '4'},
-                 'robocop-5': {'group': 'RC', 'code': '5'},
-                 'robocop-6': {'group': 'RC', 'code': '6'},
-                 'robocop-7': {'group': 'RC', 'code': '7'},
-                 'robocop-8': {'group': 'RC', 'code': '8'},
-                 'reftest': {'group': 'R', 'code': 'R'},
-                 'androidx86-set-4': {'group': 'S', 'code': '4'}}
+    response = requests.get('http://alertmanager.allizom.org/data/jobnames/', headers={'accept-encoding': 'json'}, verify=True)
+    data = json.loads(response.content)
```

Comment:
requests has a .json() method, so you don't need to explicitly make json.loads
```
data = requests.get('http://alertmanager.allizom.org/data/jobnames/', headers={'accept-encoding': 'json'}, verify=True).json()
```
should work.